package.json

```diff
@@ -61,11 +62,14 @@
     "babel-plugin-transform-flow-strip-types": "^6.8.0",
     "babel-preset-es2015": "^6.14.0",
     "babel-register": "^6.14.0",
+    "compat-table": "github:kangax/compat-table#gh-pages",
```

Comment:
Makes sense. It is still blocked by us having to run the build script to generate the json so we can at least view that beforehand

So if we don't auto run `npm run build-data`  and commit it it can be reviewed